CHANGELOG.md

```diff
@@ -4,6 +4,97 @@ All notable changes to Zebra are documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [Zebra 1.0.0-beta.14](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.0-beta.13) - 2022-08-30
+
+This release contains a variety of CI fixes, test fixes and dependency updates.
+
+### Added
+
+- Store history trees by height in the non-finalized state (#4928)
+- Breaking: Add `start` as default subcommand for `zebrad` (#4957)
+
+### Changed
+
+- Revert deserializing stored transactions in a rayon thread (#4933)
+- Update block hash checkpoints for mainnet (#4919)
+- Enable a `tinyvec` feature to speed up compilation (#4796)
+- Split the `zebra_state::service::read` module (#4827)
+- Disallow Orchard `ivk = 0` on `IncomingViewingKey::from` & `SpendingKey` generation (#3962)
+
+#### Docs
+
+- Increase disk and network requirements for long-term deployment (#4948, #4963)
```

Comment:
This will be a breaking change for some users.

Reply:
I'd say this will be a breaking change in every release from now on.

Reply:
I said 300 GB disk and 100 GB download per month because it's about enough for the next 6-12 months.